etna/lib/etna/clients/metis/client.rb

```diff
@@ -47,6 +48,45 @@ def find(find_request)
           @etna_client.bucket_find(find_request.to_h))
       end
 
+      def download_file(file_or_url = File.new, &block)
```

Comment:
Now support upload / download endspoints in the etna metis client!  Makes automation a lot easier.
In the future, I think we can consolidate and make the metis_client just depend on the etna gem and use this client directly.

Advantage is that we can re-use this logic in automation, and consolidate on a single implementation that gets lots of active testing.